### DISCUSSION TITLE
File uploading support

### DISCUSSION THREAD

**Discussion by Tryfciu** [dedoc/scramble#147] [Ideas]
Hello,

is there any chance that file uploading will be supported?

**Comment by romalytvynenko** [comment]
@Tryfciu hey! Can you please share the code examples that you have? Like the code you have, the actual result, and the expected result.

Thanks!

**Reply by Tryfciu** [reply]
I'm basing on Open API documentation: https://swagger.io/docs/specification/describing-request-body/file-upload
It proposes following schema for file uploading:
```
requestBody:
        content:
          multipart/form-data:
            schema:
              type: object
              properties:
                fileId:
                  type: string
                file:
                  type: string
                  format: binary
```

Therefore this schema should be generated by following request:

![image](https://github.com/dedoc/scramble/assets/28263270/746b0682-4c22-469e-8f8e-1315b7b1cf55)

**Comment by nanumichael27** [comment]
> Hello,
> 
> is there any chance that file uploading will be supported?

Honestly we need this

**Reply by romalytvynenko** [reply]
@nanumichael27 it is fixed already, try the latest Scramble version
